services/checks/checks_test.go

```diff
@@ -28,7 +28,7 @@ import (
 	"github.com/percona/pmm/version"
 	promtest "github.com/prometheus/client_golang/prometheus/testutil"
 	"github.com/stretchr/testify/assert"
-	"github.com/stretchr/testify/mock"
+	mock "github.com/stretchr/testify/mock"
```

Comment:
Any reason to make this?

Reply:
I actually don't remember making that alias, maybe vscode did it :thinking:

Reply:
removed it